### DISCUSSION TITLE
When will SNS OAuth2 be available?

### DISCUSSION THREAD

**Discussion by jinserk** [fief-dev/fief#23] [Q&A]
It's awesome project! BTW I am interested in Google, Facebook and other 3rd party SNS based OAuth2 to try this into my project. Do you have any timeline to implement the 3rd party OAuth2?

**Comment by frankie567** [comment]
Hi @jinserk, welcome to Fief 👋 

It's definitely on our roadmap! Currently, we are busy with roles/permissions implementation, but it's probably the next feature we'll tackle. So I would say maybe in a month or two 😄

**Reply by jinserk** [reply]
Sounds awesome! Thank you for sharing a great work!
